docs/content/interacting/transactional-writes.mdx

```diff
@@ -167,7 +167,7 @@ curl -X POST 'http://localhost:8080/stores/{store_id}/write' \
 
 </details>
 
-The Write API allows you to send up to 100 unique tuples in the request. (This limit applies to the sum of both writes and deletes in that request). This means we can submit one API call that converts the `tweet` from public to visible to only the `user`'s followers.
+By default, the Write API allows you to send up to 100 unique tuples in the request (This limit applies to the sum of both writes and deletes in that request). This means we can submit one API call that converts the `tweet` from public to visible to only the `user`'s followers. This limit can be configured on the server by specifying the [`--max-tuples-per-write`](../getting-started/configuration.mdx#OPENFGA_MAX_TUPLES_PER_WRITE) config value).
```

Comment:
At the end of the paragraph, you have a closing ")" parenthesis without a matching opening one. 

A potential fix to this could be: 

```
By default, the Write API allows you to send up to 100 unique tuples in the request (this limit applies to the sum of both writes and deletes and can be configured via [`--max-tuples-per-write`](../getting-started/setup-openfga/configuration.mdx#OPENFGA_MAX_TUPLES_PER_WRITE)). This means we can submit one API call that converts the `tweet` from public to visible to only the `user`'s followers.
```